app/src/main/java/org/lineageos/tv/launcher/MainActivity.kt

```diff
@@ -32,7 +32,6 @@ import org.lineageos.tv.launcher.utils.Suggestions.orderSuggestions
 class MainActivity : FragmentActivity(R.layout.activity_main) {
     // Views
     private val mainVerticalGridView by lazy { findViewById<VerticalGridView>(R.id.main_vertical_grid) }
-    private val settingButton by lazy { findViewById<ImageButton>(R.id.settings_button) }
```

Comment:
Eh, we usually leave views there even if used only once, like view bindings

Reply:
Yeah keep that here, it's fine